packages/blocks/src/store/test/selectors.js

```diff
@@ -86,72 +86,74 @@ describe( 'selectors', () => {
 
 	describe( 'getChildBlockNames', () => {
 		it( 'should return an empty array if state is empty', () => {
-			const state = {};
+			const state = {
+				blockTypes: {},
```

Comment:
`blockTypes` will always be there as it's part of the reducer:

https://github.com/WordPress/gutenberg/blob/3284f343a4c69a74a1e29eac03a14d0d2c6c9cc1/packages/blocks/src/store/reducer.js#L297